node/impl/full/dummy.go

```diff
@@ -0,0 +1,109 @@
+package full
+
+import (
+	"context"
+	"errors"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/chain/types/ethtypes"
+)
+
+var ErrImplementMe = errors.New("Not implemented yet")
+
+type EthModuleDummy struct{}
```

Comment:
This should be removed (it was removed in the eth_ gateway PR, not sure why it ressurected itself, but probably when merge conflicts were getting fixed)

Reply:
We're using it again since this PR https://github.com/filecoin-project/lotus/pull/10072